raspberry-pi/4/modesetting.nix

```diff
@@ -27,6 +27,13 @@ in
   };
 
   config = lib.mkIf cfg.enable {
+    # fixes a crash: https://github.com/raspberrypi/linux/issues/5568
+    # can be removed for >= nixos 23.11: https://github.com/NixOS/nixpkgs/pull/247826
+    boot.kernelParams = [ "kunit.enable=0" ];
+
+    # doesn't work for the CM module, so we exclude e.g. bcm2711-rpi-cm4.dts
```

Comment:
The glob below doesn't match this filename, and might exclude other modules, so the comment is a bit misleading.

Reply:
what do you mean? it _excludes_ the said device tree, so that is exactly the point

Reply:
The comment says we will exclude `bcm2711-rpi-cm4.dts`.

The glob says we will exclude `bcm2711-rpi-4*.dtb`.

The glob `bcm2711-rpi-4*.dtb` does not match the filename `bcm2711-rpi-cm4.dts`, so the comment is confusing at best.

Reply:
Am I right in thinking that we don’t have a Sentry project for production?

Reply:
Oh hang on, I was assuming the `.filter` _removes_ the specified files — does it instead specify which to include?

Reply:
Yes, it is an include

Reply:
Doh! Got it, thank you!